tools/cli/go-whisk-cli/commands/activation.go

```diff
@@ -101,14 +101,22 @@ var activationListCmd = &cobra.Command{
 }
 
 var activationGetCmd = &cobra.Command{
-    Use:   "get ACTIVATION_ID [FIELD_FILTER]",
+    Use:   "get (ACTIVATION_ID | --last) [FIELD_FILTER]",
     Short: wski18n.T("get activation"),
     SilenceUsage:   true,
     SilenceErrors:  true,
     PreRunE: setupClientConfig,
     RunE: func(cmd *cobra.Command, args []string) error {
         var field string
+        var err error
 
+        if args, err = lastFlag(args); err != nil {  // Checks if any errors occured in lastFlag(args)
+          whisk.Debug(whisk.DbgError, "client.Activation.Get(%#v) failed: %s\n", args, err)
```

Comment:
Debug message should contain `lastFlag` instead of `client.Activation.Get` since `lastFlag` is the method you are calling, and allows us to trace the code execution path through debug messages. Same applies to the other instances were `lastFlag` is called below.